example/static_images.js

```diff
@@ -88,14 +88,23 @@ $(function() {
                 length: 10
             },
             decoder : {
-                readers : ["code_128_reader"]
+                readers : ["code_128_reader"],
+                drawScanline: true,
+            },
+            numOfWorkers: 4,
+            locator: {
+                halfSample: false,
+                showSkeleton: true,
+                showFoundPatches: true
```

Comment:
These two settings are for debugging only (when ``showCanvas`` is set to ``true``)